src/Authentication/AzureArtifacts.cs

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) Microsoft. All rights reserved.
+//
+// Licensed under the MIT license.
+
+using System.Runtime.InteropServices;
+using Microsoft.Extensions.Logging;
+using Microsoft.Identity.Client;
+using Microsoft.Identity.Client.Broker;
+
+namespace Microsoft.Artifacts.Authentication;
+
+public static class AzureArtifacts
+{
+    public const string ClientId = "d5a56ea4-7369-46b8-a538-c370805301bf";
+
+    public static PublicClientApplicationBuilder CreateDefaultBuilder(Uri authority, ILogger logger)
```

Comment:
The logger is not used in the method - do we need it here?

Reply:
Well, we used to need it, which I realize is not a great answer. Will remove.

Reply:
reminder to remove from readme example to match as well